.github/workflows/build_agents_playground.yml

```diff
@@ -0,0 +1,79 @@
+name: Build @fiberplane/agents
+
+on:
+  pull_request:
+    branches: ["*"]
+    paths:
+      - "packages/agents/**"
+      - "packages/agents-ui/**"
+  push:
+    branches: ["main", "release-*"]
+    paths:
+      - "packages/agents/**"
+      - "packages/agents-ui/**"
+
+env:
+  CARGO_TERM_COLOR: always
+  FORCE_COLOR: true
+
+jobs:
+  build_packages:
+    name: Build packages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          run_install: false
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+          cache: "pnpm"
+
+      - name: Setup Biome
+        uses: biomejs/setup-biome@v2
+        with:
+          version: 1.9.4
+
+      - name: Install dependencies
+        run: pnpm install
+        env:
+          CI: true
+
+      # Linting: we use global biome command
+      # any extra commands should be added to the lint:ci script
+      # in the package.json
+
+      - name: Lint all workspaces (biome)
+        run: biome ci .
+
+      - name: Lint all workspaces (eslint or other linters)
+        run: pnpm --recursive lint:ci
```

Comment:
This doesn't seem to do anything (at least not for me locally):
``` bash
$ pnpm --recursive lint:ci
Scope: 8 of 9 workspace projects
None of the selected packages has a "lint:ci" script
```

it would be good to add something like `tsc --noEmit` as a "lint:ci" script